node-data/nodes.json

```diff
@@ -61,7 +61,7 @@ onLoadMarkers(
     "lon": -79.868306,
     "name": "Bell's Market",
     "address": "603 Braddock Ave., Braddock, PA 15104",
-    "hood": "Braddock",
+    "hood": "Braddock",g
```

Comment:
Oh. thanks


On Sun, May 1, 2016 at 6:36 PM, Colin Dean <notifications@github.com> wrote:

> In node-data/nodes.json
> <https://github.com/pittmesh/pittmesh.net/pull/28#discussion_r61694365>:
>
> > @@ -61,7 +61,7 @@ onLoadMarkers(
> >      "lon": -79.868306,
> >      "name": "Bell's Market",
> >      "address": "603 Braddock Ave., Braddock, PA 15104",
> > -    "hood": "Braddock",
> > +    "hood": "Braddock",g
>
> spurious g
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/pittmesh/pittmesh.net/pull/28/files/9626bd35b9d04d2e76138d1738142351c5a82abe#r61694365>
>